app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -470,11 +493,16 @@ public void onLoadFinished(@NonNull final Loader<Attachment> loader, @Nullable f
         @Override
         public Loader<Translation> onCreateLoader(final int id, @Nullable final Bundle args) {
             switch (id) {
-                case LOADER_LATEST_TRANSLATION:
+                case LOADER_LATEST_PRIMARY_TRANSLATION:
                     if (mToolCode != null) {
                         return new LatestTranslationLoader(requireContext(), mToolCode, getPrimaryLanguage());
                     }
                     break;
+                case LOADER_LATEST_PARALLEL_TRANSLATION:
+                    if (mToolCode != null && getParallelLanguage() != null) {
+                        return new LatestTranslationLoader(requireContext(), mToolCode, getParallelLanguage());
+                    }
```

Comment:
you should always create the loader even if the parallel language isn't currently set (I believe)

Reply:
hmm, you can't at the moment because locale is marked as needing to be non-null